fluent-syntax/tests/serializer_fixtures.rs

```diff
@@ -0,0 +1,68 @@
+use fluent_syntax::ast::{Entry, Resource};
+use glob::glob;
+use std::ffi::OsStr;
+use std::fs;
+use std::path::Path;
+
+use fluent_syntax::parser::parse;
+use fluent_syntax::serializer::{serialize, serialize_with_options, Options};
+
+/// List of files that currently do not roundtrip correctly.
+///
+/// - `multiline_values.ftl`: `key12` is parsed differently if indented.
+const BLACKLIST: [&str; 1] = ["multiline_values.ftl"];
```

Comment:
Suggestion: Please change this to something like `IGNORE_LIST` for a more inclusive term.